UnderscoreTests/Strings/UnderscoreStringsSpec.m

```diff
@@ -198,17 +207,32 @@
     it(@"should behave with multiple words", ^{
         expect(Underscore.string(@"hello underscore strings").split(@" ").unwrap).to.equal((@[ @"hello", @"underscore", @"strings" ]));
         expect(Underscore.split(@"hello underscore strings", @" ")).to.equal((@[ @"hello", @"underscore", @"strings" ]));
-        expect(Underscore.string(@" hello  underscore strings ").split(@" ").unwrap).to.equal((@[ @"", @"hello", @"", @"underscore", @"strings", @"" ]));
+        expect(Underscore.string(@" hello  underscore strings ")
+               .split(@" ")
+               .unwrap).to.equal((@[ @"", @"hello", @"", @"underscore", @"strings", @"" ]));
         expect(Underscore.split(@" hello  underscore strings ", @" ")).to.equal((@[ @"", @"hello", @"", @"underscore", @"strings", @"" ]));
     });
     
     it(@"should behave with diferent separators", ^{
         expect(Underscore.string(@"hello underscore strings").split(@"n").unwrap).to.equal((@[ @"hello u", @"derscore stri", @"gs"]));
         expect(Underscore.split(@"hello underscore strings", @"n")).to.equal((@[ @"hello u", @"derscore stri", @"gs"]));
-        expect(Underscore.string(@"It was the best of times, it was the worst of times").split(@"t").unwrap).to.equal((@[ @"I", @" was ", @"he bes", @" of ", @"imes, i", @" was ", @"he wors", @" of ", @"imes"]));
-        expect(Underscore.split(@"It was the best of times, it was the worst of times", @"t")).to.equal((@[ @"I", @" was ", @"he bes", @" of ", @"imes, i", @" was ", @"he wors", @" of ", @"imes"]));
-        expect(Underscore.string(@"hey this is another string hey yeah").split(@"hey").unwrap).to.equal((@[ @"", @" this is another string ", @" yeah" ]));
-        expect(Underscore.split(@"hey this is another string hey yeah", @"hey")).to.equal((@[ @"", @" this is another string ", @" yeah" ]));
+        expect(Underscore.string(@"It was the best of times, it was the worst of times")
+               .split(@"t")
+               .unwrap).to.equal((@[ @"I", @" was ", @"he bes", @" of ", @"imes, i", @" was ", @"he wors", @" of ", @"imes"]));
+        expect(Underscore.split(@"It was the best of times, it was the worst of times"
+                                , @"t")).to.equal((@[ @"I",
+                                                      @" was ",
+                                                      @"he bes",
+                                                      @" of ",
+                                                      @"imes, i",
+                                                      @" was ",
+                                                      @"he wors",
+                                                      @" of ",
+                                                      @"imes"]));
```

Comment:
This is still kinda hard to parse, maybe we should just use a shorter string to split?